lib/alephant/lookup/lookup_helper.rb

```diff
@@ -10,8 +11,9 @@ class LookupHelper
 
       attr_reader :lookup_table
 
-      def initialize(lookup_table)
+      def initialize(lookup_table, config)
         @lookup_table = lookup_table
+        @config = config
```

Comment:
I would add an `attr_reader :config` and then modify any latter calls to `@config` to `config`. Nicer to read.

Ps, you've referenced `@lookup_table` in some places and that already has a reader method so can you change those to be just `lookup_table`. Again, I just feel it looks nicer